pandas_gbq/gbq.py

```diff
@@ -1057,7 +1063,7 @@ def to_gbq(
                 DeprecationWarning,
                 stacklevel=2,
             )
-        elif api_method == "load_csv":
+        else:
```

Comment:
There's already a check above for known `api_methods`, so this was an impossible branch.